server/service/integration_mdm_dep_test.go

```diff
@@ -2134,6 +2134,7 @@ func (s *integrationMDMTestSuite) createTeamDeviceForSetupExperienceWithProfileS
 func (s *integrationMDMTestSuite) TestSetupExperienceFlowWithSoftwareAndScriptAutoRelease() {
 	t := s.T()
 	ctx := context.Background()
+	s.setSkipWorkerJobs(t)
```

Comment:
My guess is that this is what caused the issue, the worker ran implicitly when the test was taking a loooong time to run (with `-race` on probably, as is the case in the reported issue: https://github.com/fleetdm/fleet/actions/runs/15697865604/job/44226394790#step:5:1).

And that implicit run added some profile or command to the list of expected ones. Would still be nice to see which one that is, and handle it properly if/when it shows up, and that's the second part of this PR, where the debug logging is left enabled so we can have a better look of what command gets added.